support-files/mariadb.service.in

```diff
@@ -68,20 +68,23 @@ ProtectSystem=full
 # Prevent accessing /home, /root and /run/user
 ProtectHome=true
 
-# Execute pre and post scripts as root, otherwise it does it as User=
-PermissionsStartOnly=true
+# Use an environment file to pass variable _WSREP_NEW_CLUSTER
+EnvironmentFile=-@mysqlunixdir@/wsrep-new-cluster-%N
```

Comment:
`%N` would refer to a templated systemd service which is `mariadb@.service.in`. Would it make sense to include it there. I'm thinking remove it here and put it in `mariadb@.service.in`.

If that's true then, the `wsrep-start-position` would need a `-%N` suffix too to avoid conflicts in the same template file.

Reply:
`%N` is a specifier by systemd and evaluates to *Full unit name*, see [systemd.unit(5)](https://www.freedesktop.org/software/systemd/man/latest/systemd.unit.html) and search for it.

When I started this (back in 2018?) Galera cluster was supported from instantiated services. Probably we can drop this now...

Reply:
Dropped specifier in rebase.